docs/src/Components/Dropdown/Dropdown.purs

```diff
@@ -82,29 +82,26 @@ component =
 
         C.ItemSelected item -> do
           H.liftAff $ log ("Selected: " <> item)
-          H.modify \st -> st { selected = ( item : st.selected ) }
+          H.modify \st -> st { selected = pure item }
 
           st <- H.get
           H.liftAff $ logShow st.selected
-          _  <- H.query unit
-                  $ H.action
-                  $ C.ContainerReceiver
-                  $ { render: renderContainer
-                    , items: difference st.items st.selected }
+          _ <- H.query unit
+                 $ H.action
+                 $ C.SetVisibility C.Off
           pure a
 
+        otherwise -> pure a
+
       Removed item a -> do
         st <- H.get
-        let newSelections = delete item st.selected
-            newItems = difference newSelections st.items
-
-        H.modify (_ { selected = newSelections })
+        H.modify (_ { selected = Nothing })
 
         _  <- H.query unit
                 $ H.action
                 $ C.ContainerReceiver
```

Comment:
Now that there's a `ReplaceItems` query, this ought to be replaced with that.

`H.query unit $ H.action $ C.ReplaceItems st.items`